### PR TITLE
Extract help and test instructions into shared docs

### LINKED ISSUE #1214: Extract track-specific help instructions from `config/exercise_readme.go.tmpl`
Each track needs a file that contains track-specific instructions on how to get help. The contents of this document are only presented to the student when using the CLI. This file lives at `exercises/shared/.docs/help.md`. **You almost certainly already have this information, but need to move it to the correct place.**

For v2 tracks, this information was (usually) included in the readme template found at `config/exercise_readme.go.tmpl`. As such, tracks can extract the help instructions from the `config/exercise_readme.go.tmpl` file to the `exercises/shared/.docs/help.md` file.

See https://github.com/exercism/csharp/pull/1557/files for an example PR.

# Tracking

https://github.com/exercism/v3-launch/issues/50

### LINKED ISSUE #1215: Extract track-specific test instructions from `config/exercise_readme.go.tmpl`
Each track needs a file that contains track-specific instructions on how to manually run the tests. The contents of this document are only presented to the student when using the CLI. This file lives at `exercises/shared/.docs/tests.md`. **You almost certainly already have this information, but need to move it to the correct place.**

For v2 tracks, this information was (usually) included in the readme template found at `config/exercise_readme.go.tmpl`. As such, tracks can extract the test instructions from the `config/exercise_readme.go.tmpl` file to the `exercises/shared/.docs/tests.md` file.

See https://github.com/exercism/csharp/pull/1557/files for an example PR.

# Tracking

https://github.com/exercism/v3-launch/issues/51

### PR DESCRIPTION
**Pull request by tejasbubane**
Closes #1214
Closes #1215

### CODE REVIEW

File: exercises/shared/.docs/tests.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+```bash
+# Using npm
+npm test
+
+# Alternatively using yarn
+yarn test
+```
+
+In the test suites all tests but the first have been skipped.
```
Comment (SleeplessByte) [inline exercises/shared/.docs/tests.md:32]: In some* test suites

**Review by SleeplessByte — APPROVED** [review]

File: exercises/shared/.docs/tests.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Tests
+
+## Setup
+
+Go through the setup [instructions for JavaScript][docs-exercism-javascript] to install the necessary dependencies.
```
Comment (junedev) [inline exercises/shared/.docs/tests.md:5]: The link seems to be missing. Was this intended?

**Comment by junedev** [comment]
@tejasbubane Thanks a lot for taking care of this!

File: exercises/shared/.docs/tests.md (hunk 2 above)
Comment (tejasbubane) [reply exercises/shared/.docs/tests.md:5]: Not intended. I missed it while moving this line from `cli.md`. Thanks for spotting :+1:

**Pull request merged**